### DISCUSSION TITLE
[Feature Request] Remove frutrating "white-space" when using element with different background inside a <v-navigation-drawer>

### DISCUSSION THREAD

**Discussion by plumedev** [vuetifyjs/vuetify#20426] [Q&A]
### Problem to solve
When your using a `<v-navigation-draw>` (white background) with E.g. a `<v-sheet>` (with black background) you'll always see a frustrating white space. You can watch a demo in this [vuetify playground](https://play.vuetifyjs.com/#eNqdVMtu4zAM/BXB59Wm2Nch6AYosMf9g/UeaImx1cqSINFOgqL/Xlp+1G7SS32wLZqeGVIc/XsuUlS7hxC+9h0W++KesA0WCA+lE+K+lwqizu95ZeHiOxKJLhZ/l0UF6qmOvnN6LyrLi7KYcnO2g97UQMY7qSOcMAplIaXhx1qeGkNYFoIabAcsDZF/FwFjCw4dLUAZKjWI9BYRC5KWR4tn8dglMseLTAEUygrphOgEWFM7qRiNuUOSP0W4yO9lsQHy1seF/+3Lip8VaNMvlGTIoiA801jE+Nr8Gp9d4BIUJBRH7zgDTd2QrLzVq+aMVwba0OyYZ134bqp82wxrEi26KYLjqiMX+Y5gypQssd0Sh4gBnZZGeccQrTayN3iSGlJTed7xbYsmpZz553bCYRC6cH1eBCjFwzR06/yRgocx5VP8tyhr6NEOYzjj69Y4buRHiHOcQ6vg7BrB+2U97YGHwM2+uTFCAeS3q3FgqRU5NpJXTwfx19eD1QY+jm6hrsZk49pZ5pX91t5swbjFx00e0r34cXcXuPO5+CFhNv5Qc3b+eCrwcjoWNszFyxfXWZtv/18Bf5VRjg==)

### Proposed solution
There is no explicit CSS which specify border, outline or anything esle which can make this white space.

 But you can fix this by specifing a `border-width: 0px` by default by two ways: 
 1. Inside de vuetify.ts by putting this style by default: 
`VNavigationDrawer: {style: {borderWidth:  '0px'}}`
1. With a custom CSS (not best)
`.v-navigation-drawer { border-width: 0px !important; }`

I'm pretty sure, Vuetify team can fix this by puttin this by default (of course, if a custom css add a border or a border-0 utility class it will be overrided.

**Comment by KaelWD** [comment]
https://vuetifyjs.com/en/api/v-navigation-drawer/#props-floating
https://vuetifyjs.com/en/api/v-navigation-drawer/#sass-navigation-drawer-border-thin-width
